### PR TITLE
Fix #2 Migrated to Github Actions from Gitlabs CI.

### LINKED ISSUE #2: Migrate from GitLab CI to GitHub Actions
Now that GitHub has built-in CI/CD, it'd be easier to use GitHub Actions than the current [GitLab CI](https://github.com/neelkamath/seeds/blob/master/.gitlab-ci.yml) setup.

### PR DESCRIPTION
**Pull request by Sharkaboi**
Let me know if I need to change anything :)

### CODE REVIEW

File: .github/tests.yml
Comment (neelkamath) [inline .github/tests.yml]: Tests usually get run on other branches as well because sometimes a contributor/maintainer would want to verify it works before merging, or because other branches are used for different editions of the software. So we should probably keep this to `on: push` instead.

File: .github/tests.yml
Comment (neelkamath) [inline .github/tests.yml]: The `gradlew` is meant for direct running, and as a result, already has the permissions set. We can remove this step.

**Review by neelkamath — CHANGES_REQUESTED** [review]

File: .github/tests.yml
Comment (Sharkaboi) [reply .github/tests.yml]: Thanks for the nit, It has become a habit as I work on windows haha. 
Sending Updated commit now :)

**Review by neelkamath — APPROVED** [review]

**Pull request merged**
